analytics_queries/induction_tutors.sql

```diff
@@ -0,0 +1,21 @@
+\copy (
+  SELECT s.urn,
+          (icp.id IS NOT NULL)               AS tutor_nominated,
+          icp.created_at                     AS tutor_nominated_at,
+          (u.current_sign_in_at IS NOT NULL) AS tutor_signed_in,
+          u.full_name                        AS tutor_name,
+          u.email                            AS tutor_email,
+          (pp.id IS NOT NULL)                AS sit_mentor,
+          u.id                               AS sit_mentor_id
```

Comment:
Just wondering if this needs to be the external identifier from the participant identity or not here?

Reply:
Good call, Nathan agrees. I'll switch this over before merging.

Reply:
@tonyheadford - ok switched this over, would you mind taking another quick look?